src/core/Messages.ts

```diff
@@ -33,6 +33,10 @@ export interface ErrorMessage {
     message: string;
 }
 
+export function isErrorMessage(msg: any): boolean {
+    return !!msg.message && !!msg.message.message && typeof msg.message.message === 'string';
```

Comment:
Basically, you should only have `!!msg.message` here (and maybe the typecheck, but the typescript compiler is supposed to make such checks less critical, so I'm inclined to leave it out too). Reason being that by this stage, all messages should be standardised, and must thus conform to the `ErrorMessage` interface above.

If you're seeing cases where this isn't happening, it should be flagged as a bug.